bin/ci-builder

```diff
@@ -134,6 +134,15 @@ case "$cmd" in
         docker push materialize/ci-builder:"$cache_tag"
         ;;
     run)
+        if [[ $1 == "--detach" ]]; then
```

Comment:
I think examples of more principled parsing using while + case can be found here:

https://stackoverflow.com/questions/192249/how-do-i-parse-command-line-arguments-in-bash

the answer has 3475 likes , wow

This way the order of `--detach` vs `--name` does not matter and they can be specified independently if needed.

Reply:
✅